infrastructure/main.tf

```diff
@@ -0,0 +1,145 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+
+  required_version = ">= 1.0.0"
+}
+
+provider "aws" {
+  shared_credentials_files = ["~/.aws/credentials"]
+  profile                  = "igork_aws"
+  default_tags {
+    tags = {
+      Owner = "Igor Kostin"
+    }
+  }
+}
+
+locals {
+  subnet_prefix_name = {
+    public  = "public_SUBENT_${var.owner_code_name}_${var.vpc_version}"
+    private = "private_SUBENT_${var.owner_code_name}_${var.vpc_version}"
+    db      = "db_SUBENT_${var.owner_code_name}_${var.vpc_version}"
+  }
+
+  subnet_cider_bits = {
```

Comment:
No need for this at all as it's not dynamic, you are not doing iteration over this, it can be manually put into your "public_subnet_cider_blocks" module.